From 2651a41985773e8150b3602b0af892f06ec81033 Mon Sep 17 00:00:00 2001
From: Michael Walle <michael@walle.cc>
Date: Thu, 2 Sep 2021 15:42:01 +0200
Subject: [PATCH 28/32] arm: dts: ls1028a: drop non-removable property from
 esdhc controller node

The linux device tree hasn't set this property. To be similarly we remove
it from the u-boot device tree, too. The second controller of the
LS1028A has indeed no card detect pin. The present state register of the
second controller will always report the card as present. Thus, there
should be no functional change otherwise than one more register access
to read the present state. The property should still be introduced in
the linux device tree.

Signed-off-by: Michael Walle <michael@walle.cc>
---
 arch/arm/dts/fsl-ls1028a.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/dts/fsl-ls1028a.dtsi b/arch/arm/dts/fsl-ls1028a.dtsi
index d2f558d208..af6dabe847 100644
--- a/arch/arm/dts/fsl-ls1028a.dtsi
+++ b/arch/arm/dts/fsl-ls1028a.dtsi
@@ -210,7 +210,6 @@
 			reg = <0x0 0x2150000 0x0 0x10000>;
 			interrupts = <GIC_SPI 63 IRQ_TYPE_LEVEL_HIGH>;
 			big-endian;
-			non-removable;
 			bus-width = <4>;
 			status = "disabled";
 		};
-- 
2.30.2

